pkg/util/github/github.go

```diff
@@ -99,3 +102,59 @@ func NewClient(option *Option) (*Client, error) {
 
 	return client, nil
 }
+
+func FetchRemoteContent(ctx context.Context, owner string, repo string, path string, wsFiles []string) ([]string, map[string]error, error) {
+	var filesInRemoteDir = make([]string, 0)
+	_, dirContent, resp, err := client.Repositories.GetContents(
```

Comment:
The client is `nil` here. You shouldn't use the `client` directly here, but put this code into a client method, something like: 

```go
func (c *Client) FetchRemoteContent(){c.XXX()}
```